core/src/main/java/org/apache/accumulo/fate/AdminUtil.java

```diff
@@ -293,12 +292,10 @@ private void findLocks(ZooReader zk, final String lockPath,
 
             if (pos == 0) {
               locks = heldLocks;
+            } else if (lda[0].charAt(0) == 'R' && !sawWriteLock) {
+              locks = heldLocks;
             } else {
-              if (lda[0].charAt(0) == 'R' && !sawWriteLock) {
-                locks = heldLocks;
-              } else {
-                locks = waitingLocks;
-              }
+              locks = waitingLocks;
```

Comment:
This was converted by my editor as redundant when I saved the file, and is clearly better, so I kept it.